packages/p2p/src/components/orders/order-details/order-details-chatbox.jsx

```diff
@@ -0,0 +1,97 @@
+import classNames from 'classnames';
+import * as React from 'react';
+import { Loading } from '@deriv/components';
+import { Channel, SendBirdProvider } from 'sendbird-uikit';
+import Dp2pContext from 'Components/context/dp2p-context';
+// import { getShortNickname, generateHexColourFromNickname } from 'Utils/string';
+import 'sendbird-uikit/dist/index.css';
+
+const OrderDetailsChatbox = ({ token, app_id, user_id, channel_url }) => {
+    const [is_loading, setIsLoading] = React.useState(true);
+    const { is_dark_mode_on } = React.useContext(Dp2pContext);
+
+    React.useEffect(() => {
+        const interval_header = setInterval(() => {
+            const el_sendbird_conversation = document.querySelector('.sendbird-conversation');
+            const el_chat_title = document.querySelector('.sendbird-chat-header__title');
+            // const el_chat_avatars = document.getElementsByClassName('sendbird-avatar');
+
+            if (el_chat_title && el_chat_title.innerText !== 'No title') {
+                clearInterval(interval_header);
+                setIsLoading(false);
+                el_sendbird_conversation.setAttribute('style', 'display: flex;');
+            } else {
+                return;
+            }
+
+            // if (el_chat_avatars) {
```

Comment:
can remove this comments, since it wont be used anymore

Reply:
We will use this once we have access to the counterparty's details.